packages/@sanity/cli/vitest.config.mts

```diff
@@ -8,8 +8,11 @@ dotenv.config({
 
 export default defineConfig({
   test: {
-    includeSource: ['./test/**/*.ts'],
+    includeSource: ['./test/**/*.ts', './src/**/__tests__/**/*.ts'],
     globalSetup: ['./test/shared/globalSetup.ts'],
     testTimeout: 30000, // 30s
   },
+  define: {
+    __DEV__: JSON.stringify(true),
+  },
```

Comment:
Necessary to get the config loader to work in tests – it's checking for `__DEV__`